sdcm/nemesis.py

```diff
@@ -1357,7 +1356,7 @@ def disrupt_nodetool_flush_and_reshard_on_kubernetes(self):
             raise UnsupportedNemesis('It is supported only on kubernetes')
         dc_idx = 0
         for node in self.cluster.nodes:
-            if hasattr(node.k8s_cluster, 'eks_cluster_version') and node.scylla_shards >= 14:
+            if hasattr(node.k8s_cluster, 'eks_cluster_version') and node.scylla_shards >= 7:
```

Comment:
You might want to update the comments below

Reply:
Done